iota/develop/sidebars.ts

```diff
@@ -101,6 +101,23 @@ module.exports = {
         },
       ],
     },
+    {
+      type: 'category',
+      label: 'Endpoints',
```

Comment:
As we also mention websites like the explorer and faucet, would something like `Endpoints & Developer Infrastructure` fit better? Any suggestions? Or should we just leave it?

Reply:
How about `Endpoints and Developer Tools` ?

Reply:
But I want to get rid of that name 😅 . The pages does't really contain tools. And we already have a "Tools" section in the sidebar

Reply:
I'd remove the whole Developer Tools section and move faucet endpoint up right after the MQTT endpoints. We need a bit of explanation that this specific net provides a faucet to be able to get tokens to work with, provide the endpoint and then point to how to actually use the endpoint.

Reply:
And move the rest of the tools in Developer Tools to the Tools section already provided in the sidebar